srcpkgs/mpg123-extras/template

```diff
@@ -13,11 +14,9 @@ short_desc="Fast console MPEG audio decoder/player - extra plugins"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="LGPL-2.1-only"
 homepage="https://www.mpg123.org/"
-changelog="https://www.mpg123.org/cgi-bin/scm/mpg123/trunk/NEWS"
```

Comment:
ftr, this link doesnt work

Reply:
Yeah, that's why I changed it to something that does.

Or do you mean something different?

Reply:
oh it was deleted >_<